README.md

```diff
@@ -1,2 +1,26 @@
 # HW01
 Edit the readme
+#this is an edit made by Cassandra
+
+# Biography of an MA gal
+It is a *myth* that all of us have a NE accent, sadly, but not so sadly, I don't think I have it. However, the occasional word sounds a bit NE which makes me feel the past 8 years have not been for **naught**.
```

Comment:
Cool. Looking forward to hearing it. I've been in California like my whole life, so this will be interesting.